actions/universal-description-quicklinks.js

```diff
@@ -87,10 +103,10 @@ module.exports = (course, item, callback) => {
         } else {
             //checking to see if a link meetings the criteria
             $(links).each((i, link) => {
-                if (typeof $(link).attr('href') != "undefined" &&
-                    $(link).attr('href') != null) {
+                if ($(link).attr('href') !== undefined &&
```

Comment:
This can be switched to just check if it is falsy. Both `null` and `undefined` are falsy.

Reply:
I'm pretty sure I did this and it isn't showing up until you edit it...